AppDevAnnouncements/Classes/NotificationView.swift

```diff
@@ -0,0 +1,103 @@
+//
+//  NotificationView.swift
+//  AppDevAnnouncements
+//
+//  Created by Gonzalo Gonzalez on 10/16/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class NotificationView: UIView {
+
+    // Components
+    var notificationContainerView: UIView!
+    var dismissButton: UIButton!
+    var visualImageView: UIImageView!
+    var subjectLabel: UILabel!
+    var bodyTextView: UITextView!
+    var ctaButton: UIButton!
```

Comment:
Can we do 
```swift
let notificationContainerView = UIView()
let dismissButton = UIButton()
...
``` 
instead so we can't overwrite those objects

Reply:
Also lets make them private and just pass in necessary information in a custom `init`

Reply:
I made everything in NotificationViewController private too and passed what needed to be changed into a new convenience init inside NotificationView.